### DISCUSSION TITLE
Add labels for each color step (index)

### DISCUSSION THREAD

**Discussion by bryanberger** [primer/prism#32] [Done]
![Screen Shot 2022-06-17 at 8 27 31 AM](https://user-images.githubusercontent.com/940022/174298171-71a2314e-c007-496b-8847-e5c8f735a538.png)

I find myself getting a bit lost in our palettes (we have 20+ steps) highlighting the index value atop each color would be helpful imo. I've got this running locally can get a PR up if interested.

**Comment by colebemis** [comment]
Great idea. I'd be happy to review a PR for this 👍

**Reply by bryanberger** [reply]
Opened a PR here — https://github.com/primer/prism/pull/40

**Comment by colebemis** [comment]
FYI, I'm converting this into a discussion in the "Ideas" category

**Comment by colebemis** [comment]
Thank you for implementing this @bryanberger! 💖  https://github.com/primer/prism/pull/40
